packages/manager/apps/web/client/app/components/manager-order-catalog-price/manager-order-catalog-price.controller.js

```diff
@@ -25,12 +26,10 @@ export default class {
   }
 
   getPriceText(priceInCents) {
+    const locale = this.TranslateService.getUserLocale().replace('_', '-');
```

Comment:
Maybe we could add a `translate` function to `TranslateService` so we don't have to use `$translate` everywhere 🤔

Reply:
I don't understand your proposition. You want to use `TranslateService.translate('translation_key')` instead of `$translate.instant('translation_key')` ?
What are the benefits of this ?

Reply:
````javascript
TranslateService.translate('translation_key')
````

will never change as we have control on it, whereas

````javascript
$translate.instant('translation_key')
````

will change the day we get rid of AngularJS/Pascal Precht to


````javascript
$t('translation_key')
````

the day we use `vue-18n` or 



````javascript
t('translation_key')
````

the day we use `react-i18next`

Reply:
`TranslateService` is currently specific to angular so it will still imply a few changes. Moreover what about the `translate` directives ? It would require a rewrite.
But it would provide abstraction from the used library

Reply:
`TranslateService` is just a name, you could inject anything in it 🙂 

I don't have an answer for `translate` directives - except of course that @cbourgois while codemod the heck out of them, all I know it's that it has not much to do with not using `$translate` anymore